### PR TITLE
[IMP] theme_kea: update the theme

### PR DESCRIPTION
**Pull request by xlu-odoo**
This commit update the theme to improve the design
with background shapes, new pictures and new custom snippets.

Task-2591476

### CODE REVIEW

File: theme_kea/static/src/js/tour.js [hunk 1]
```diff
@@ -8,9 +8,13 @@ var tour = require("web_tour.tour");
 
 const snippets = [
     {
-        id: 's_cover',
+        id: 's_cover', //0
```
Comment (Cocographique) [inline theme_kea/static/src/js/tour.js:11]: Unnecessary diff.
Remove //0

File: theme_kea/static/src/scss/primary_variables.scss [hunk 2]
```diff
@@ -1,77 +1,20 @@
-//------------------------------------------------------------------------------
```
Comment (Cocographique) [inline theme_kea/static/src/scss/primary_variables.scss:1]: Keep this part for compatibility purposes.

File: theme_kea/static/src/scss/primary_variables.scss [hunk 3]
```diff
@@ -166,9 +115,10 @@ $o-theme-color-palettes-compatibility-indexes: (
 // Shapes
 //------------------------------------------------------------------------------
 
-$o-bg-shapes: change-shape-colors-mapping('web_editor', 'Origins/03', (3: 5));
```
Comment (Cocographique) [inline theme_kea/static/src/scss/primary_variables.scss:169]: Same as **Real Estate**: keep for compatibility?

File: theme_kea/views/snippets/s_cover.xml [hunk 4]
```diff
@@ -3,23 +3,32 @@
     <template id="s_cover" inherit_id="website.s_cover">
         <!-- Section -->
         <xpath expr="//section" position="attributes">
-            <attribute name="class" add="o_full_screen_height" separator=" "/>
-            <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/07_002","flip":[]}</attribute>
+            <attribute name="class" add="s_parallax_no_overflow_hidden oe_img_bg o_full_screen_height" remove="parallax s_parallax_is_fixed bg-black-50 remove" separator=" "/>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_cover.xml:6]: Why do you remove `.remove`? 😄 
I think you are trying to remove the parallax effect here. So why do you add `.s_parallax_no_overflow_hidden`?

File: theme_kea/views/snippets/s_cover.xml [hunk 5]
```diff
@@ -3,23 +3,32 @@
...
+            <attribute name="data-scroll-background-ratio">0</attribute>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Floats/02","flip":[]}</attribute>
+            <attribute name="style">background-image: url('/web/image/website.s_cover_default_image'); background-position: 50% 0%;</attribute>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_cover.xml:9]: `background-image` and `background-position`
Same as the snippet. This line change nothing.

File: theme_kea/views/snippets/s_image_gallery.xml [hunk 6]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="s_images_wall" inherit_id="website.s_images_wall">
+        <!-- Section -->
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="pt96 pb96" remove="pt24 pb24" separator=" "/>
+        </xpath>
+        <!-- Replace images -->
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_image_gallery.xml:8]: Same as **Real Estate**

File: theme_kea/views/snippets/s_media_list.xml [hunk 7]
```diff
@@ -3,51 +3,67 @@
...
-            <attribute name="class" add="o_cc5 flex-row-reverse rounded" remove="o_cc1" separator=" "/>
-            <attribute name="style" add="border-radius: 25px !important;" separator=" "/>
+            <attribute name="class" remove="o_cc1" separator=" "/>
         </xpath>
         <xpath expr="(//div[hasclass('col-lg-4')])[3]" position="attributes">
-            <attribute name="class" add="col-lg-3" remove="col-lg-4" separator=" "/>
+            <attribute name="class" add="col-lg-6" remove="col-lg-4" separator=" "/>
+        </xpath>
+        <xpath expr="(//img)[3]" position="attributes">
+            <attribute name="style" add="border-radius:200px !important" separator=" "/>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_media_list.xml:21]: This should be an option available in the editor with other image style options (rounded, square, thumbnail...).
In BS, there is a class and a variable that we could use:
```scss
$rounded-pill: 50rem !default;

.rounded-pill {
    border-radius: $rounded-pill !important;
}

File: theme_kea/views/snippets/s_title.xml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="s_title" inherit_id="website.s_title">
+        <!-- Section -->
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="o_cc2 pb40 pt56" remove="pt40 pb40" separator=" "/>
+        </xpath>
+        <!-- Title -->
+        <xpath expr="//h1" position="replace">
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_title.xml:9]: Same as **Real Estate**

File: theme_kea/views/snippets/s_picture.xml [hunk 9]
```diff
@@ -3,22 +3,31 @@
...
         <xpath expr="//h2" position="replace">
             <h2 style="text-align: center;">
-                <font style="font-size: 62px;">Don’t miss a frame</font>
+                <font style="font-size: 62px;">Experience a new world</font>
             </h2>
         </xpath>
+        <!-- Image -->
+        <xpath expr="//img" position="attributes">
+            <attribute name="class" add="rounded-lg" remove="padding-large" separator=" "/>
+            <attribute name="style" add="border-radius:500px !important; min-height: 500px;" separator=" "/>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_picture.xml:22]: `border-radius: 500px;`

File: theme_kea/views/snippets/s_picture.xml [hunk 10]
```diff
@@ -3,22 +3,31 @@
...
+        </xpath>
         <!-- Paragraphs -->
         <xpath expr="//p[1]" position="replace">
-            <p class="text-center">Our new drone enable transmission up to 15 km away and supports 4K videos.<br/>
-            Keep the contact, even in a tropical storm.</p>
+            <p class="text-center">Put yourself at the center of an extraordinary gaming universe with exclusive games.</p>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_picture.xml:26]: Use `style="text-align: center;"` instead, in order to make it work with the editor.

File: theme_kea/views/snippets/s_image_text.xml [hunk 11]
```diff
@@ -3,19 +3,24 @@
...
+            <attribute name="src">/web_editor/shape/theme_kea/s_image_text.svg?c1=%23d46926</attribute>
         </xpath>
         <!-- Title & Paragraph -->
         <xpath expr="(//div[hasclass('col-lg-6')])[2]" position="replace">
             <div class="col-lg-6 pt16 pb16">
-                <h2>Enter in a new world of control.</h2>
-                <p>Control every electronic device of your house. The coffee machine, the television, the thermostat, everything.</p>
-            </div>
+                <h2>Enter in a new world</h2>
+                    <p>Write one or two paragraphs describing your product or services. To be successful your content needs to be useful to your readers.</p>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_image_text.xml:21]: Don't put back the default text. Just remove the previous customization.
No need to replace the whole column if you just want to replace the title.

File: theme_kea/views/snippets/s_color_blocks_2.xml [hunk 12]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="s_color_blocks_2" inherit_id="website.s_color_blocks_2">
+        <!-- Box #1 -->
+        <xpath expr="//div[hasclass('col-lg-6')]" position="attributes">
+            <attribute name="class" add="oe_img_bg o_bg_img_center pt256 pb256" remove="" separator=" " />
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_color_blocks_2.xml:6]: Remove `remove=""` if you don't use it.

File: theme_kea/views/snippets/s_cover.xml [hunk 13]
```diff
@@ -3,23 +3,32 @@
...
-        <xpath expr="//section//*[last()]" position="after">
-            <div class="o_we_shape o_web_editor_Origins_07_002"/>
+        <xpath expr="//div[hasclass('container')]" position="before">
+            <div class="o_we_shape o_web_editor_Floats_02"/>
+        </xpath>
+        <!-- Disable Parallax -->
+        <xpath expr="//span[hasclass('s_parallax_bg')]" position="replace"/>
+        <!-- Title -->
+        <xpath expr="//h1" position="replace">
+            <h1 class="text-center"><span style="font-size: 62px;">The Revolution <br/>in Virtual Reality</span></h1>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_cover.xml:23]: Use style="text-align: center;" instead, in order to make it work with the editor.

File: theme_kea/views/snippets/s_cover.xml [hunk 14]
```diff
@@ -3,23 +3,32 @@
...
+        </xpath>
+        <!-- Paragraph -->
+        <xpath expr="//p[hasclass('lead')]" position="replace">
+            <p class="lead"><br/></p>
+        </xpath>
+        <!-- Button -->
+        <xpath expr="//a[hasclass('btn')]" position="replace">
+            <a href="#0" class="btn btn-lg btn-primary">Discover more</a>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_cover.xml:31]: `#0` ?

File: theme_kea/views/snippets/s_color_blocks_2.xml [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+            <a href="#" class="btn btn-link text-white mt-2 font-weight-bold">Explore more ></a>
+        </xpath>
+
+        <!-- Box #2 -->
+        <xpath expr="//div[hasclass('col-lg-6')][2]" position="attributes">
+            <attribute name="class" add="oe_img_bg o_bg_img_center pt256 pb256" remove="" separator=" " />
+            <attribute name="style">background-image: url('/web/image/website.s_color_blocks_2_default_image'); background-position: 50% 0%;</attribute>
+        </xpath>
+        <!-- Box #2 - Icon -->
+        <xpath expr="//div[hasclass('col-lg-6')][2]//i" position="replace">
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_color_blocks_2.xml:34]: When there is nothing to replace, use a self-closing tag.

File: theme_kea/views/snippets/s_color_blocks_2.xml [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+        </xpath>
+        <!-- Box #1 - Text -->
+        <xpath expr="//div[hasclass('col-lg-6')]//p" position="replace">
+            <p>Conquer leaderboards, crush workouts, create with friends and more. Get lost in a growing library of VR classics and all-new experiences.</p>
+        </xpath>                
+        <!-- Box #1 - Button -->
+        <xpath expr="//div[hasclass('col-lg-6')]//a" position="attributes">
+            <attribute name="class" add="btn-secondary" remove="btn-primary" separator=" " />
+        </xpath>
+        <xpath expr="//div[hasclass('col-lg-6')]//a" position="replace">
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_color_blocks_2.xml:24]: Same xpath as above.
If you want a text to be bold, wrap it with <b></b>, otherwise, it won't work with the editor.
Do you really need to add `.text-white`?

Same problems in Box #2

File: theme_kea/views/snippets/s_media_list.xml [hunk 17]
```diff
@@ -3,51 +3,67 @@
...
         <xpath expr="(//div[hasclass('row')])[2]" position="attributes">
-            <attribute name="class" add="rounded flex-row-reverse o_cc5" remove="o_cc1" separator=" "/>
-            <attribute name="style" add="border-radius: 25px !important;" separator=" "/>
+            <attribute name="class" remove="o_cc1" separator=" "/>
         </xpath>
         <xpath expr="//div[hasclass('col-lg-4')]" position="attributes">
             <attribute name="class" add="col-lg-6" remove="col-lg-4" separator=" "/>
         </xpath>
+        <xpath expr="//img" position="attributes">
+            <attribute name="style" add="border-radius:200px !important" separator=" "/>
```
Comment (Cocographique) [inline theme_kea/views/snippets/s_media_list.xml:44]: `border-radius:200px;`

File: theme_kea/views/snippets/s_cover.xml (hunk 5 above)
Comment (xlu-odoo) [reply theme_kea/views/snippets/s_cover.xml:9]: I think I need to keep this line (9) because I'm "disabling" the parallax span a bit later. Usually the `background-image` is on the `span `and not `section`, that's probably why I put it there. Without this line, the background image doesn't display at all.

File: theme_kea/views/snippets/s_cover.xml (hunk 14 above)
Comment (xlu-odoo) [reply theme_kea/views/snippets/s_cover.xml:31]: Keeping this `#0` so it doesn't scroll to top of page.

File: theme_kea/static/src/scss/primary_variables.scss [hunk 18]
```diff
@@ -109,21 +52,27 @@ $o-theme-font-configs: (
...
-        'header-template': 'default',
-        'footer-template': 'call_to_action',
+        'btn-border-radius': 10rem,
+        // Header
+        'header-links-style': 'pills',
+        'header-template': 'Contact',
+        // Footer
+        'footer-template':                  'minimalist',
+        'footer-effect':                    'slideout_slide_hover',
+        'footer-scrolltop':                 true,
```
Comment (qsm-odoo) [inline theme_kea/static/src/scss/primary_variables.scss:75]: Only one space between `:` and the value please

File: theme_kea/static/src/scss/primary_variables.scss [hunk 19]
```diff
@@ -109,21 +52,27 @@ $o-theme-font-configs: (
...
+        'buttons-font': 'Poppins',
+        // Links / Buttons
         'link-underline': 'never',
-        'header-links-style': 'outline',
-        'btn-primary-outline': true,
-        'header-template': 'default',
-        'footer-template': 'call_to_action',
+        'btn-border-radius': 10rem,
+        // Header
+        'header-links-style': 'pills',
```
Comment (qsm-odoo) [inline theme_kea/static/src/scss/primary_variables.scss:70]: The `'pills'` style must have the `header_navbar_pills_style` enabled to work and I don't see you enabling it, how can this work? :thinking:

File: theme_kea/static/src/scss/primary_variables.scss [hunk 20]
```diff
@@ -109,21 +52,27 @@ $o-theme-font-configs: (
...
+        'header-template': 'Contact',
```
Comment (qsm-odoo) [inline theme_kea/static/src/scss/primary_variables.scss:71]: Same here, you set header-template to `'Contact'` but I don't see you enabling the contact header view, what am I missing?

File: theme_kea/views/snippets/s_color_blocks_2.xml [hunk 21]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="s_color_blocks_2" inherit_id="website.s_color_blocks_2">
```
Comment (qsm-odoo) [inline theme_kea/views/snippets/s_color_blocks_2.xml:3]: For all new files, try to follow our correct convention: no indentation and a blank line after odoo

```xml
<odoo>

<template>
[...]
</template>

</odoo>
```

File: theme_kea/views/snippets/s_cover.xml [hunk 22]
```diff
@@ -3,23 +3,32 @@
...
+        <!-- Update Filter -->
+        <xpath expr="//div[hasclass('o_we_bg_filter')]" position="replace">
+            <div class="o_we_bg_filter" style="background-color: rgba(0, 0, 0, 0.10)"/>
         </xpath>
         <!-- Shape -->
-        <xpath expr="//section//*[last()]" position="after">
-            <div class="o_we_shape o_web_editor_Origins_07_002"/>
+        <xpath expr="//div[hasclass('container')]" position="before">
+            <div class="o_we_shape o_web_editor_Floats_02"/>
+        </xpath>
```
Comment (qsm-odoo) [inline theme_kea/views/snippets/s_cover.xml:18]: Since you replace the filter in the xpath just above, could you not add the shape div at the same time? That would avoid an xpath for it, so better perf, less risk to have it broken at some point, less code, etc.

File: theme_kea/views/snippets/s_media_list.xml [hunk 23]
```diff
@@ -3,52 +3,78 @@
...
+        </xpath>
+        <xpath expr="//div[hasclass('s_media_list_body')]" position="attributes">
+            <attribute name="class" add="col-lg-6" remove="col-lg-8" separator=" "/>
         </xpath>
-        <xpath expr="(//div[hasclass('col-lg-4')])[3]" position="attributes">
-            <attribute name="class" add="col-lg-3" remove="col-lg-4" separator=" "/>
+        <xpath expr="(//img)" position="attributes">
+            <attribute name="src">/web_editor/image_shape/website.s_media_list_default_image_3/web_editor/basic/bsc_organic_2.svg</attribute>
+            <attribute name="data-shape">web_editor/basic/bsc_organic_2</attribute>
+            <attribute name="data-shape-colors">;;;;</attribute>
```
Comment (qsm-odoo) [inline theme_kea/views/snippets/s_media_list.xml:26]: Missing shape attributes as discussed (at multiple locations in this PR)

File: theme_kea/static/description/index.html [hunk 24]
```diff
@@ -1,203 +1 @@
-<section class="container">
-    <h2 class="text-muted">Html5 Responsive Bootstrap Theme for Odoo CMS</h2>
```
Comment (qsm-odoo) [inline theme_kea/static/description/index.html:2]: In this commit message, I see `[REM] theme_real_estate: remove outdated theme description` :thinking:

**Review by qsm-odoo — CHANGES_REQUESTED** [review]
@xlu-odoo I rebased the branch on 14.5, push-forced and retargeted the PR. When you push-force on your side, make sure you are rebased on it too.

Here are some comments for the kea theme we'd like to merge today. Also, I see some images are yet too heavy <500kb :grimacing: Could you check that as well? 

+ commit message convention: task-xxx instead of Task-xxx.

Also, ensure the IAP PR is up-to-date too and mention it in the kea task so we can merge them together. Thanks :+1:

File: theme_paptic/__manifest__.py [hunk 25]
```diff
@@ -4,7 +4,7 @@
     'category': 'Theme/Corporate',
     'summary': 'Consultancy, Design, Tech, Computers, IT, Blogs',
     'sequence': 110,
-    'version': '2.0.0',
+    'version': '2.1.0',
```
Comment (qsm-odoo) [inline theme_paptic/__manifest__.py:7]: @xlu-odoo I am not sure what you did but you have a master commit of Brieuc in your branch :/ Did you fetch the branch I rebased like I warned you about and/or rebased it on 14.5 by yourself?

File: theme_kea/models/theme_kea.py [hunk 26]
```diff
@@ -6,7 +6,7 @@ class ThemeKea(models.AbstractModel):
 
     def _theme_kea_post_copy(self, mod):
         self.disable_view('website.footer_custom')
-        self.enable_view('website.template_footer_call_to_action')
-
+        self.enable_view('website.template_footer_minimalist')
+
```
Comment (qsm-odoo) [inline theme_kea/models/theme_kea.py:10]: Configure your editor correctly and review your code before sending it to me: no spaces at the end of lines please, thanks :+1:

File: theme_kea/static/src/scss/primary_variables.scss (hunk 2 above)
Comment (qsm-odoo) [reply theme_kea/static/src/scss/primary_variables.scss:1]: @xlu-odoo You still not handled that before asking for review today? :thinking:

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

**Comment by qsm-odoo** [comment]
@xlu-odoo If you can also send the IAP update PR (here or in the task) so I can merge it too, thanks :+1:

**Review by qsm-odoo — APPROVED** [review]
@robodoo rebase-ff r+
